### PR TITLE
Feature/T-3: Create Home page

### PR DESCRIPTION
**Pull request by crisan-catalin**

### CODE REVIEW

File: src/main/webapp/resources/css/style.css [hunk 1]
```diff
@@ -0,0 +1,13 @@
+html, body {
+    height: 100vh;
+}
+
+.nav-logo {
+    width: 200px;
+}
+
+.homepage-img-cover {
+    background: url(/resources/images/home-background.jpg);
```
Comment (LunguNicolae) [inline src/main/webapp/resources/css/style.css:10]: Wouldn't it be better if you could use a functionality of show-hide but without putting hard coded img paths inside .css, what if we will need to reuse this class functionality in other places in our web app?

File: src/main/webapp/WEB-INF/jsp/header.jsp [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    </a>
+
+    <div class="d-flex flex-column flex-md-row">
+        <div class="align-self-center mr-md-4">
+            <a class="p-2 text-dark" href="/cart/">
+                <i class="fas fa-shopping-cart pr-md-2"></i> Checkout your order
+            </a>
+        </div>
+
+        <c:if test="${empty sessionUser.name}">
```
Comment (LunguNicolae) [inline src/main/webapp/WEB-INF/jsp/header.jsp]: maybe it would be better if you could use  <c:choose> tag instead <c:if>

Comment (crisan-catalin) [reply src/main/webapp/WEB-INF/jsp/header.jsp]: Done

File: src/main/webapp/resources/css/style.css (hunk 1 above)
Comment (crisan-catalin) [reply src/main/webapp/resources/css/style.css:10]: This is a specific class for homepage image use. I renamed it with a more verbose name.

**Pull request merged**